src/app/modules/main/communities/controller.nim

```diff
@@ -398,3 +403,8 @@ proc authenticateWithCallback*(self: Controller) =
 
 proc getCommunityPublicKeyFromPrivateKey*(self: Controller, communityPrivateKey: string): string =
   result = self.communityService.getCommunityPublicKeyFromPrivateKey(communityPrivateKey)
+
+proc asyncCheckPermissionsToJoin*(self: Controller, communityId: string, addressesToShare: seq[string]) =
+  self.tmpCommunityId = communityId
+  self.tmpAddressesToShare = addressesToShare
+  self.communityService.asyncCheckPermissionsToJoin(communityId)
```

Comment:
You're not passing `addressesToShare` here, so you're just checking the basic addresses